packages/okta-react/src/AuthService.js

```diff
@@ -59,9 +58,11 @@ class AuthService {
     this.emit = this.emit.bind(this);
     this.on = this.on.bind(this);
 
-    this._subscriberCount = 0;
-
     this.clearAuthState();
+
+    // Remove expired token or renew token if autoRenew is true
+    // This process will keep authState synced with states in tokenManager
+    this.on('expired', this.updateAuthState);
```

Comment:
@swiftone The real fix is here. Right now react sdk only listen on `onSessionExpired` callback, which only handled session expiration but not token expiration. 
auth-js has mechanism to set up an expiration timer after retrieval of a new token, and emit `expired` event when token is about to expire by tracking `expireAt` field from token. 

The change here is to listen on that expired event, then renew token from the tokenManager.get.